datadog_lambda/extension.py

```diff
@@ -1,5 +1,5 @@
 import logging
-from os import path
+import os
```

Comment:
why importing the whole`os` instead of `path` from `os`?

Reply:
habit. I'm just used to seeing `os.path.exists` not `path.exists`.